Dockerfile

```diff
@@ -0,0 +1,34 @@
+# The builder image, used to build the virtual environment
+FROM python:3.11-buster as builder
+
+RUN apt-get update && apt-get install -y git
+
+RUN pip install poetry==1.4.2
+
+ENV POETRY_NO_INTERACTION=1 \
+    POETRY_VIRTUALENVS_IN_PROJECT=1 \
+    POETRY_VIRTUALENVS_CREATE=1 \
+    POETRY_CACHE_DIR=/tmp/poetry_cache
+
+ENV HOST=0.0.0.0
+ENV LISTEN_PORT 8051
+EXPOSE 8051
+
+WORKDIR /app
```

Comment:
Do these get picked up in the runtime image as well?

Reply:
well spotted. I have removed the above lines except WORKDIR where ENV is created and COPIED to runtime